content/Arduino Cloud/General Cloud/Import-an-Arduino-Cloud-Template.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Import an Arduino Cloud Template"
+---
+
+In this article, you'll learn the step-by-step process for importing an Arduino Cloud template.
+
+## Import a Template
+
+1. Find a [Template](https://app.arduino.cc/templates) and click "Import Template" (If you have issues importing the template, you can find more information [here](https://support.arduino.cc/hc/en-us/articles/4408887422994-If-you-can-t-import-a-template-in-Arduino-Cloud))
+
+1. Plug in your device and wait for it to be detected. Once the device is detected, click on 'Configure' to continue.
+
+    ![Dialog box displaying that the device was found](img/Device_found.png)
+
+1. Now that your device is detected and connected, the firmware of your Arduino device will be checked and updated if necessary to enhance connectivity. Follow the on-screen steps to proceed.
+
+    ![Dialog box displaying firmware updates](img/Checking_device_connectivity.png)
+
+1. Once the device setup is complete, enter your network credentials.
+
+    ![Dialog box for entering network credentials](img/Set_network_parameters.png)
```

Comment:
```suggestion
    ![Dialog box for entering network credentials](img/Set_network_parameters.png)
    
---
    
## Still need help?

* [If you can't import a template in Arduino Cloud](https://support.arduino.cc/hc/en-us/articles/4408887422994-If-you-can-t-import-a-template-in-Arduino-Cloud)
* [If your device can't be added to Arduino Cloud](https://support.arduino.cc/hc/en-us/articles/360019355679-If-your-device-can-t-be-added-to-Arduino-Cloud)
```
Added section with links to troubleshooting articles relevant during the process